libraries/AP_HAL_ChibiOS/hwdef/EAH743QF/hwdef.dat

```diff
@@ -0,0 +1,172 @@
+# hw definition file for processing by chibios_pins.py
+# for Ewing Aerospace H743QF hardware.
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID for firmware load
+APJ_BOARD_ID AP_HW_EAH743QF
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+FLASH_SIZE_KB 2048
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 128
+
+# only one I2C bus
+I2C_ORDER I2C1
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 UART5 USART6
+
+# Buzzer - DMA timer channel use by LEDs
+PC13 BUZZER OUTPUT GPIO(80) LOW
+define HAL_BUZZER_PIN 80
+
+
+
+# USB
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# Debug
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+# SPI1 for ICM42688
+PA4 ICM42688_CS CS
+PA5 SPI1_SCK SPI1
+PA6 SPI1_MISO SPI1
+PA7 SPI1_MOSI SPI1
+
+# SPI2 for MAX7456
+PB12 OSD_CS CS
+PB13 SPI2_SCK SPI2
+PB14 SPI2_MISO SPI2
+PB15 SPI2_MOSI SPI2
+
+# SPI3 for W25Q128JVS
+PA15 FLASH_CS CS
+PC10 SPI3_SCK SPI3
+PC11 SPI3_MISO SPI3
+PB2  SPI3_MOSI SPI3
+
+# I2C1 for Baro_BMP390
+PB8 I2C1_SCL I2C1 PULLUP
+PB9 I2C1_SDA I2C1 PULLUP
+
+PC0 BATT_VOLTAGE_SENS ADC1 SCALE(1) ANALOG(10)
+PC1 BATT_CURRENT_SENS ADC1 SCALE(1) ANALOG(11)
+
+# define default battery setup
+define HAL_BATT_VOLT_PIN 10
+define HAL_BATT_CURR_PIN 11
+define HAL_BATT_VOLT_SCALE 11.1
+define HAL_BATT_CURR_SCALE 100
+define HAL_BATT_MONITOR_DEFAULT 4
+
+# USART1 - GPS
+PB7  USART1_RX USART1
+PB6  USART1_TX USART1
+define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_GPS
+
+
+# USART2 - OSD HD
+PA2  USART2_TX USART2
+PA3  USART2_RX USART2
+define DEFAULT_SERIAL2_PROTOCOL SerialProtocol_MSP_DisplayPort
+
+# USART3 - OSD Analog
+PB11 USART3_RX USART3
+PB10 USART3_TX USART3
+define SERIAL3_DEFAULT_PROTOCOL SerialProtocol_SmartAudio
+
+# UART4 - SIK (GCS)
+PA0  UART4_TX UART4 NODMA
+PA1  UART4_RX UART4 NODMA
+define DEFAULT_SERIAL4_PROTOCOL SerialProtocol_MAVLink2
+
+# UART5
+PD2  UART5_RX UART5 NODMA
+PC12 UART5_TX UART5 NODMA
+
+# USART6 - RX-only from HD_VTX
+PC7  USART6_RX USART6 NODMA
```

Comment:
rc should have dma, but recheck resolverr for TIM1 qnd 3 after to be sure its not shared